frontend/survey-project/src/app/shared/services/auth.service.ts

```diff
@@ -10,9 +16,47 @@ export class AuthService {
 
     constructor(
         private readonly cookieService: CookieService,
+        private readonly router: Router,
+        private readonly userService: UserService
+
     ) {}
 
     loginUser(session: Session): void {
         this.cookieService.set('ssid', session.id, new Date(session.ending_time));
     }
+
+    logoutUser(): void {
+    
+        this.userService.logoutUser().subscribe({
```

Comment:
In this whole service, move every . to be in the new line (also this should apply to all of the .subscribing in the project.
this.userService
      .logoutUser()
      .subscribe(
...
);